tdrs-frontend/zap-scanner.sh

```diff
@@ -8,8 +8,8 @@ fi
 
 
 
-docker-compose down
-docker-compose up -d --build
+docker-compose -f docker-compose.yml down
```

Comment:
@carltonsmith Checking I have this right - since we explicitly specify a `docker-compose.yml` file here, the `docker-compose.override.yml` is not used in this case, so the `.env` file variables are also never used - is that correct?

Reply:
Yes, since the Zap Scanner in CircleCi doesn't have the .env file it was failing without forcing it to only use the one .yml file